sos/report/plugins/postfix.py

```diff
@@ -52,6 +54,38 @@ def forbidden_ssl_keys_files(self):
         finally:
             return fp
 
+    def forbidden_password_files(self):
+        forbid_attributes = (
+            "lmtp_sasl_password_maps",
+            "smtp_sasl_password_maps",
+            "postscreen_dnsbl_reply_map",
+            "smtp_sasl_auth_cache_name",
+        )
+        fp = []
+        prefix = 'hash:'
+        option_format = re.compile(r"^(.*)=(.*)")
+        try:
+            with open(self.path_join('/etc/postfix/main.cf'), 'r') as cffile:
+                for line in cffile.readlines():
+                    # ignore comment and check option format
+                    line = re.sub('#.*', '', line)
```

Comment:
It makes sense to scrub passwords also in comments, no? Two use cases:

1) Sometimes, previously used passwords are kept in comments as a kind of history. Even the historic passwords are worth to obfuscate, since they might be reused or still in use else-where (yes, neither that is a good password policy, but I have seen that in practise)

2) Sometimes, for debugging purposes, one can temporarily comment out existing password to test a password-less authentication.

Reply:
I think it may be a different issue from excluding files.
In my humble opinion, If necessary, we should create a new issue.

Reply:
Ah I apologize for my mental shortcut (we usually scrub passwords using regexp, while here we stop collecting files using regexp). I meant "stop collecting forbidden files mentioned also in comments". But I agree there would be less valid use cases for it, so I am OK with the current PR.